src/map/entities/mobentity.cpp

```diff
@@ -603,7 +603,14 @@ void CMobEntity::Spawn()
         }
     }
     
-    m_DespawnTimer = time_point::min();
```

Comment:
Perhaps I'm misunderstanding but..This mobmod is for NMs that stay up for so many seconds after the alliance wipes. And anything flagged as an NM should already ignores the regular depop timer even without this mobmod set at all. So the conditional check here should not be required.

As far as bigMobMod...I was shocked to look here and see `milliseconds` when the normal value is in seconds  (300 for a mob that will stay up at its ??? spot for 5 min before depopping).